flask-backend/api/schema/management.py

```diff
@@ -10,4 +10,4 @@ class ManagementSchema(ma.Schema):
     Schema for admin model
     """
     class Meta:
-        fields = ('name', 'email', 'public_id', 'varified', 'created_on', 'updated_on', 'role')
\ No newline at end of file
+        fields = ('name', 'email', 'public_id', 'vefified', 'created_on', 'updated_on', 'role')
```

Comment:
Check here!

Reply:
My apologies for letting this typo slide through. I'll fix this right away.